packages/studio/styled-system/jsx/is-valid-prop.mjs

```diff
@@ -436,7 +436,7 @@ var userGenerated = [
   '_light',
   '_osDark',
   '_osLight',
-  '_highContrast',
+  '_highConstrast',
```

Comment:
idk what happened here... I didn't change this

Reply:
this is a change from a few days ago, this should have been updated with it, it's fine